### PR TITLE
[DRAFT] [Issue #4325] List of documentation improvements

### PR DESCRIPTION
**Pull request by afabiani**
To build the docs locally, go to geonode/docs folder and run

`sphinx-build -T -E -d _build/doctrees-readthedocs -D language=en . _build/html`

You will find the compiled docs under `geonode/docs/_build/html/index.html`

### CODE REVIEW

**Comment by t-book** [comment]
@afabiani I´m a bit puzzled. Why did this merge  https://github.com/GeoNode/geonode/commit/834ef7133a948b8bccd7d9ce51b8cdf7736e37d6  seemingly trigger this build https://readthedocs.org/projects/geonode/builds/8889435/ which published this draft commit with your changes to settings docs? http://docs.geonode.org/en/master/tutorials/advanced/geonode_settings/settings.html 
🤔
Maybe this makes more sense by tomorrow with fresh eyes. I will go through the settings file then as well.

File: docs/tutorials/advanced/geonode_settings/settings.txt [hunk 1]
```diff
@@ -23,47 +23,80 @@ A
...
+
+    When a user logs into GeoNode, if no ``ACCESS_TOKEN`` exists, a new one will be created with a default expiration time of ``ACCESS_TOKEN_EXPIRE_SECONDS`` seconds (1 day by default).
 
 * ACCOUNT_APPROVAL_REQUIRED
 ---------------------------
 
-    TBD
+    Default: ``False``
+
+    Env: ``ACCESS_TOKEN_EXPIRE_SECONDS``
```
Comment (giohappy) [inline docs/tutorials/advanced/geonode_settings/settings.txt]: this shoulb be ACCOUNT_APPROVAL_REQUIRED

File: docs/tutorials/advanced/geonode_settings/settings.txt [hunk 2]
```diff
@@ -23,47 +23,80 @@ A
...
+
+    If ``ACCESS_TOKEN_EXPIRE_SECONDS`` equals ``True``, new registered users must be activated by a superuser through the Admin gui, before they can access GeoNode.
```
Comment (giohappy) [inline docs/tutorials/advanced/geonode_settings/settings.txt]: same here

**Comment by afabiani** [comment]
@t-book I run some builds manually, but also I started the doc refactoring with the previous PR already.

**Comment by t-book** [comment]
@afabiani sorry for noise. I was wrong yesterday. After my comment I´ve recognized that it was a prev PR. I will come back soon with my review. Thanks!

**Comment by afabiani** [comment]
@t-book sure

**Comment by t-book** [comment]
@afabiani I think we could merge this with the little changes requested by @giohappy and add more work later? Let me know if I can help with something.

**Comment by afabiani** [comment]
@t-book agree. I'll take care of this today hopefully.

**Pull request merged**